backend/contactsAPI.py

```diff
@@ -39,6 +43,28 @@ def get_contacts():
         return Response(status=204)
 
 
+@app.route('/img', methods=['POST'])
+def image():
+    if request.method == 'POST':
+        token = request.headers.get('token')
+        _id = request.form.get('_id')
+        user = User().find_by_token(token)
+        image = request.files['file']
+        hashedName = image.split(".", 1)[0] + str(hash(_id)) + str(hash(datetime.now().strftime("%d")))
```

Comment:
`image` is not of a type that can be split, when I try to run the code I'm getting a 500. Try `image.filename`